client/homebrew/phbStyle/phb.style.less

```diff
@@ -468,3 +468,99 @@ body {
 		margin-bottom : 10px;
 	}
 }
+//*****************************
+// *      MAGIC ITEM CARD
+// *****************************/
+.phb .card {
+	width: var(--card-width);
+	height: var(--card-height);
```

Comment:
I suggest we set some default value here so users won't have to put `style="--card-width: 67mm; --card-height: 89mm;"` into each card. Docs for specifying the default value: [here](https://developer.mozilla.org/en-US/docs/Web/CSS/var()#Values)